receiver/cloudfoundryreceiver/receiver.go

```diff
@@ -159,3 +239,13 @@ func createLibraryMetricsSlice(metrics pmetric.Metrics) pmetric.MetricSlice {
 	libraryMetrics.Scope().SetName(instrumentationLibName)
 	return libraryMetrics.Metrics()
 }
+
+func createLibraryLogsSlice(logs plog.Logs) plog.LogRecordSlice {
+	resourceLogs := logs.ResourceLogs()
```

Comment:
Personal preference, but would it make sense to combine some of these lines? Instead of each method and append call on their own line, we could just have them together.